### PR TITLE
objective-c docs

### PR DESCRIPTION
**Pull request by theRealWardo**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -38,6 +39,22 @@ Pick which SDK you want to use from the tabs on the right (or in the nav on mobi
...
+```objc--objective_c
+// for iOS
+@import MUXSDKStats;
+
+// for tvOS
+@import MUXSDKStatsTv;
+```
+
+<p class="lang-specific objc">
+  Include the correct Mux Objective-C SDK for your project by cloning our repository and then bringing the right framework into your project. The <code>Frameworks</code> folder contains two folders for iOS and tvOS. Inside these folders, there are 3 additional folders containing different architecture combinations. The <code>fat</code> folder contains a library with all architectures in one. This is library cannot be used when compiling for submission to the App Store as it contains the simulator architectures that are not used by any Apple devices. You can use the framework in the <code>release</code> folder when building a release version of your application, or you can run <a href="https://gist.github.com/brett-stover-hs/b25947a125ff7e38e7ca#file-frameworks_blogpost_removal_script_a-sh">a script to strip unneeded architectures</a>. Finally, don't forget to add the correct import statement for your target platform.
```
Comment (jsanford8) [inline source/index.html.md]: `The Frameworks folder contains two folders for iOS and tvOS`

to

`The Frameworks folder contains two folders, one for iOS and one for tvOS`

File: source/index.html.md [hunk 2]
```diff
@@ -122,6 +139,39 @@ videojs('my-player', {
 });
 ```
 
+<p class="lang-specific objc">
+  To monitor the performance of an AVPlayer, call either <code>monitorAVPlayerViewController:withPlayerName:andConfig:</code> or <code>monitorAVPlayerLayer:withPlayerName:andConfig:</code>, passing a pointer to your AVPlayer container to the SDK. When calling <code>destroyPlayer</code> or <code>videoChangeForPlayer:withConfig:</code> to <a href="#changing-the-video">change the video</a> the same player name used for the monitor call must be used.
```
Comment (jsanford8) [inline source/index.html.md]: `passing a pointer to your AVPlayer container (either the AVPlayerLayer or AVPlayerViewController)`

**Review by jsanford8 — APPROVED** [review]

File: source/index.html.md (hunk 1 above)
Comment (theRealWardo) [reply source/index.html.md]: done

File: source/index.html.md (hunk 2 above)
Comment (theRealWardo) [reply source/index.html.md]: done

**Pull request merged**